src/us/kbase/workspace/test/controllers/handle/HandleServiceController.java

```diff
@@ -35,115 +34,118 @@
 	
 	private final Process handleService;
 	private final int handleServicePort;
-	private final Process handleManager;
-	private final int handleManagerPort;
 	
 	private final Path tempDir;
 	
-	private final static String DB = "hsi";
-	private final static String TABLE = "Handle";
-	private final static String USER = "hsitest";
-	private final static String PWD = "hsi-pass-test";
+	private final static String DB = "handle_db";
+	private final static String COLLECTION = "handle";
 	
 	private final static String HANDLE_SERVICE_NAME = "handle_service";
 
 	public HandleServiceController(
-			final String plackupExe,
-			final String abstractHandlePSGIpath,
-			final String handleManagerPSGIpath,
-			final String handleManagerAllowedUser,
-			final MySQLController mysql,
+			final MongoController mongo,
 			final String shockHost,
 			final AuthToken shockAdminToken,
 			final String perl5lib,
```

Comment:
pretty sure you don't need the perl lib location

Reply:
👍